docs/resources/compute_instance_v2.md

```diff
@@ -283,7 +283,7 @@ The following arguments are supported:
   Changing this creates a new server.
 
 * `security_groups` - (Optional) An array of one or more security group names
-  to associate with the server. Changing this results in adding/removing
+  or IDs (recommended) to associate with the server. Changing this results in adding/removing
```

Comment:
We should not recommend an approach leading to always updating not changed resource :slightly_frowning_face: